platform/ios/src/MGLMapAccessibilityElement.m

```diff
@@ -0,0 +1,217 @@
+#import "MGLMapAccessibilityElement.h"
+#import "MGLDistanceFormatter.h"
+#import "MGLCompassDirectionFormatter.h"
+#import "MGLFeature.h"
+#import "MGLVectorSource+MGLAdditions.h"
+
+#import "NSBundle+MGLAdditions.h"
+
+typedef CLLocationDegrees MGLLocationRadians;
+typedef CLLocationDirection MGLRadianDirection;
+typedef struct {
+    MGLLocationRadians latitude;
+    MGLLocationRadians longitude;
+} MGLRadianCoordinate2D;
```

Comment:
What is the difference between this struct and https://github.com/mapbox/mapbox-gl-native/blob/master/platform/darwin/src/MGLGeometry_Private.h#L17?

Reply:
Ah, I completely forgot we had implemented this in #9110. Yes, I’ll add this to the to-do list.

Reply:
Actually it was https://github.com/mapbox/mapbox-gl-native/pull/8713 😜